Makefile

```diff
@@ -366,31 +366,31 @@ test-e2e: test/instrumented-sample-app/certs/cert.pem test/instrumented-sample-a
 
 .PHONY: test-e2e-alertmanager
 test-e2e-alertmanager:
-	EXCLUDE_PROMETHEUS_TESTS=exclude EXCLUDE_PROMETHEUS_ALL_NS_TESTS=exclude EXCLUDE_THANOSRULER_TESTS=exclude EXCLUDE_OPERATOR_UPGRADE_TESTS=exclude EXCLUDE_FEATURE_GATED_TESTS=exclude EXCLUDE_PROMETHEUS_UPGRADE_TESTS=exclude $(MAKE) test-e2e
+	EXCLUDE_ALERTMANAGER_TESTS= EXCLUDE_PROMETHEUS_TESTS=exclude EXCLUDE_PROMETHEUS_ALL_NS_TESTS=exclude EXCLUDE_THANOSRULER_TESTS=exclude EXCLUDE_OPERATOR_UPGRADE_TESTS=exclude EXCLUDE_FEATURE_GATED_TESTS=exclude EXCLUDE_PROMETHEUS_UPGRADE_TESTS=exclude $(MAKE) test-e2e
```

Comment:
Did you leave `EXCLUDE_ALERTMANAGER_TESTS= ` to test if logging working?

Reply:
similarly for other test as well

Reply:
IMHO it's more explicit and easier to understand which sub test suite is executed.